### DISCUSSION TITLE
Reporting and Tracking Issues for released GitHub Features

### DISCUSSION THREAD

**Discussion by bytefish** [community/community#66464] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

You have rolled out a lot of features in the past months, such as a new code view, a new code search and lately a new dashboard feed. Some of the features have issues, that slow down work, make some tasks harder, some tasks impossible and make the life of maintainers complicated. 

At the moment people are reporting bugs in the discussions on the features. Often enough there are a dozen of discussions on a feature, and these discussions may or may not be tracked by GitHub employees. The discussions are littered with noise, spam and frustrated users. On top there are no indicators, what discussions you identify as important.

This leaves the community with a totally intransparent decision process and unable to track issues. Has a similar issue been reported? Has someone looked at my bug report at all? What's the priority of the issue? Is it being worked on? Has it already been rolled out? No one knows.

Don't get me wrong, but why can't GitHub themselves use "GitHub Issues" for bug reports, prioritizing issues and a maybe even sketch a vague roadmap for the community, so we know what's going to be worked on? There's absolutely no need to see your internal discussions, meeting notes, implementation details, commits... it would just be nice to have a central place for adressing and tracking issues.

Is anything planned to have a professional process to report issues or are you planning to stick to discussions?

**Comment by uyqadri** [comment]
Fix code search recency sort.

**Reply by le2Ks** [reply]
They will never fix it. This is how the world runs now. If something's running just fine, it's gonne be replaced by a broken one. Society been doing this with roads for decades already, now it's the same thing with programs and online platforms. Github had its days numbered, seen it coming.

**Comment by bytefish** [comment]
I am closing this discussion. It was merely shouting into the void. GitHub, as the forefront of Open Source, apparently prefers to manage bugs in Discussions.
